ledger/daml-on-sql/src/main/scala/on/sql/Cli.scala

```diff
@@ -43,6 +43,17 @@ private[sql] final class Cli(
       )
       .action((url, config) => config.copy(jdbcUrl = Some(url)))
 
+    parser
+      .opt[String]("sql-start-mode")
+      .optional()
+      .text(
+        s"The mode under which the schema SQL script will be run.  Defaults to ${DefaultConfig.sqlStartMode}.  Possible values are :" +
+          s"${PostgresStartupMode.MigrateOnly} : Run schema migration scripts and exit" +
+          s"${PostgresStartupMode.MigrateAndStart} : Run schema migration scripts and start service" +
+          s"${PostgresStartupMode.ValidateAndStart} : Validate schema migration scripts and start service"
+      )
+      .action((mode, config) => config.copy(sqlStartMode = PostgresStartupMode.fromString(mode)))
```

Comment:
Can we make these lowercase and hyphenated, to align with other parameters (and general Linux conventions)?

Something like `--sql-start-mode=migrate-and-start`, etc.

Reply:
changed and now there are only `migrate-only` and `migrate-and-start` options 

also i removed `validate-and-start` as we really cannot do that given we always run schema migrate before initializing dao for the reset behaviour stuff